toolchain/parser/parse_node_kind.def

```diff
@@ -41,49 +41,138 @@
   CARBON_PARSE_NODE_KIND(Name)
 #endif
 
-// Declarations.
-CARBON_PARSE_NODE_KIND_CHILD_COUNT(DeclarationEnd, 0)
+// The end of the file.
+CARBON_PARSE_NODE_KIND_CHILD_COUNT(FileEnd, 0)
+
+// An empty declaration, such as `;`.
 CARBON_PARSE_NODE_KIND_CHILD_COUNT(EmptyDeclaration, 0)
+
+// A name.
 CARBON_PARSE_NODE_KIND_CHILD_COUNT(DeclaredName, 0)
 
-CARBON_PARSE_NODE_KIND_CHILD_COUNT(PackageDirective, TodoFixParseNode)
+// `package`:
+//   PackageIntroducer
+//   (name)
+//     (library)
+//   PackageLibrary
+//   PackageApi or PackageImpl
+// Package
+CARBON_PARSE_NODE_KIND_CHILD_COUNT(PackageIntroducer, 0)
 CARBON_PARSE_NODE_KIND_CHILD_COUNT(PackageApi, 0)
 CARBON_PARSE_NODE_KIND_CHILD_COUNT(PackageImpl, 0)
 CARBON_PARSE_NODE_KIND_CHILD_COUNT(PackageLibrary, 1)
-CARBON_PARSE_NODE_KIND_CHILD_COUNT(PackageEnd, 0)
+CARBON_PARSE_NODE_KIND_BRACKET(Package, PackageIntroducer)
 
+// A code block:
+//   CodeBlockStart
+//   (statements)
+// CodeBlock
+CARBON_PARSE_NODE_KIND_BRACKET(CodeBlock, CodeBlockStart)
+CARBON_PARSE_NODE_KIND_CHILD_COUNT(CodeBlockStart, 0)
```

Comment:
Done, was mostly trying to get them in approximate output order, mainly because having "Start" first feels right -- but also because with things that have multiple children, it reflects the contents a little more maybe.